sources/csv/csv.go

```diff
@@ -18,6 +19,9 @@ func (t *table) start(reader *csv.Reader) {
 	defer close(t.rows)
 
 	headers, err := reader.Read()
+	if _, ok := err.(*csv.ParseError); ok {
+		log.Printf("csv.go received encoding/csv.ErrQuote, this can happen when a csv uses the wrong delimiter. Current delimiter is %q.", reader.Comma)
```

Comment:
I am concerned about not bubbling up the csv.ParseError type, I think consumers of optimus should get the lowest error we can give them, unless we're wanting to implement our own errors like optimus.sourceError or something? Also the 'extraneous "' part is itself extraneous. I'll chat with you in person.